content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,63 @@ menu:
 weight: 101
 ---
 
+## v2.0.4 General Availability [2021-02-04]
+
```

Comment:
The `Docker` section at the top of the `influxdb` version is missing, is that intentional?

Most people will probably want to use the DockerHub version once it's released, but we've had some reports of people looking for the ARM64 version of our quay.io image.

Reply:
oops! added